test/unit/middleware/prepareOutput.js

```diff
@@ -39,23 +40,70 @@ describe('prepareOutput', () => {
     sinon.assert.notCalled(next)
   })
 
-  it('calls outputFn with default options and no post* middleware (async)', () => {
+  it('calls outputFn with default options and no post* middleware and next when restify option', () => {
     let req = {
       method: 'GET',
       erm: {}
     }
 
     let options = {
+      restify: true,
       onError: onError,
-      outputFn: outputFnPromise
+      outputFn: outputFn
     }
 
     prepareOutput(options)(req, {}, next)
 
-    sinon.assert.calledOnce(outputFnPromise)
-    sinon.assert.calledWithExactly(outputFnPromise, req, {})
+    sinon.assert.calledOnce(outputFn)
+    sinon.assert.calledWithExactly(outputFn, req, {})
     sinon.assert.notCalled(onError)
-    sinon.assert.notCalled(next)
+    sinon.assert.calledOnce(next)
+  })
+
+  it('calls outputFn with default options and no post* middleware (async)', (done) => {
+    let req = {
+      method: 'GET',
+      erm: {}
+    }
+
+    let options = {
+      onError: onError,
+      outputFn: outputFnPromise
+    }
+
+    prepareOutput(options)(req, {}, next)
+
+    outputFnPromise().then(() => {
+      sinon.assert.calledTwice(outputFnPromise)
+      sinon.assert.calledWithExactly(outputFnPromise, req, {})
+      sinon.assert.notCalled(onError)
+      sinon.assert.notCalled(next)
+      done()
+    })
+
+  })
+
+  it('calls outputFn with default options and no post* middleware and next when restify option (async)', (done) => {
+    let req = {
+      method: 'GET',
+      erm: {}
+    }
+
+    let options = {
+      restify: true,
+      onError: onError,
+      outputFn: outputFnPromise
+    }
+
+    prepareOutput(options)(req, {}, next)
+
+    outputFnPromise().then(() => {
+      sinon.assert.calledTwice(outputFnPromise)
+      sinon.assert.calledWithExactly(outputFnPromise, req, {})
+      sinon.assert.notCalled(onError)
+      sinon.assert.calledOnce(next)
+      done()
+    });
```

Comment:
Codacy has a fix for the issue: [Delete `;`](https://app.codacy.com/gh/florianholzapfel/express-restify-mongoose/pullRequest?prid=8423227)
```suggestion
    })
```